audio.py

```diff
@@ -88,3 +137,20 @@ def compute_duration(audio_path, backend = 'ffmpeg'):
 	cmd = ['ffprobe', '-v', 'error', '-show_entries', 'format=duration', '-of', 'default=noprint_wrappers=1:nokey=1'
 			] if backend == 'ffmpeg' else ['soxi', '-D'] if backend == 'sox' else None
 	return float(subprocess.check_output(cmd + [audio_path]))
+
+
+if __name__ == '__main__':
+	parser = argparse.ArgumentParser()
+	parser.add_argument('--audio_path', type = str, required = True)
+	parser.add_argument('--sample-rate', type = int, default = 8000, required = True)
+	parser.add_argument('--mono', type = bool, action = 'store_true')
+	parser.add_argument('--audio-backend', type = str, required = True)
+	parser.add_argument('--show-time', type = str, action = 'store_true')
```

Comment:
There is no need to for `show-time` for the perf test, it will skew results significantly when `n_times > 0`. Printing time will dominate time spent on audio decoding.

Reply:
done